tools/code_format/check_format.py

```diff
@@ -186,6 +186,8 @@
 # Map a line transformation function across each line of a file,
 # writing the result lines as requested.
 # If there is a clang format nesting or mismatch error, return the first occurrence
+
```

Comment:
Please don't add superfluous blank lines.

Reply:
seems issues with my local env. correcting it.